docs/reference/third-party-dependencies/3_1_0.md

```diff
@@ -1,16 +1,16 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-dependencies.html
-navigation_title: Third-party dependencies 
+navigation_title: Third-party dependencies for 3.1.0
```

Comment:
because these sit under a "third-party dependencies" heading, we can trim this to just the version #

```suggestion
navigation_title: 3.1.0
```